src/TranslatorProviders.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation;
+
+use Drupal\Core\Entity\EntityTypeInterface;
+
+/**
+ * Checks if an entity type is exposed to any translator provider.
```

Comment:
Service used for determining if an entity type is exposed to our translation system.